packages/rpc-api/src/simulateTransaction.ts

```diff
@@ -117,6 +117,8 @@ type SimulateTransactionApiResponseBase = Readonly<{
     }> | null;
     /** The number of compute budget units consumed during the processing of this transaction */
     unitsConsumed?: bigint;
+    /** The number of bytes of all accounts loaded by this transaction */
+    loadedAccountsDataSize?: bigint;
```

Comment:
Since this is a u32 on the RPC side, we'll need to add this to ‘[the list of properties that are supposed to be `Number` instead of `BigInt`](https://github.com/anza-xyz/kit/blob/39662f2b2499742080616082ea527d5bc96dbbff/packages/rpc-api/src/index.ts#L345-L348),’ and then type this as `number`.